applications/SwimmingDEMApplication/python_scripts/cellular_flow/pouliot_benchmark_2D_analysis.py

```diff
@@ -1,8 +1,8 @@
 import KratosMultiphysics as Kratos
 from KratosMultiphysics import Parameters
 import KratosMultiphysics.SwimmingDEMApplication as SDEM
-import swimming_DEM_procedures as SDP
-import ethier_benchmark_analysis
+import KratosMultiphysics.SwimmingDEMApplication.swimming_DEM_procedures as SDP
+import cellular_flow.ethier_benchmark_analysis as ethier_benchmark_analysis
```

Comment:
same

Reply:
Done in next push. However it's not in the tests, maybe it's deprecated.